homeassistant/components/huawei_lte/__init__.py

```diff
@@ -186,15 +186,8 @@ def _get_data(self, key: str, func: Callable[[], Any]) -> None:
         try:
             self.data[key] = func()
         except ResponseErrorLoginRequiredException:
-            if isinstance(self.connection, AuthorizedConnection):
+            if self.connection.user_session is not None:
```

Comment:
AuthorizedConnection was deleted in the current version of the library.
Is this condition correct?